core/service-test-runner/service-tester.js

```diff
@@ -1,10 +1,11 @@
 'use strict'
 
 const emojic = require('emojic')
+const trace = require('../base-service/trace')
 const frisby = require('./icedfrisby-no-nock')(
+  // eslint-disable-next-line import/order
   require('icedfrisby-nock')(require('icedfrisby'))
```

Comment:
Interesting. I figured eslint would've recognized this automatically and not barked

Reply:
Yea, the rule probably could be improved. IMO if there are multiple requires on a line it should only consider the outermost one. 🤷‍♂️